bot/tests/test_remote.py

```diff
@@ -696,21 +694,15 @@ def test_coverity_task(
         issue.as_text()
         == f"Checker reliability is high, meaning that the false positive ratio is low.\nSome error here"
     )
-    assert issue.as_phabricator_unitresult() == UnitResult(
-        namespace="code-review",
-        name="general",
-        result=UnitResultState.Fail,
-        details=f"Code review bot found a **build error**: \n{issue.message}",
-        format="remarkup",
-    )
+
     assert check_stats(
         [
             ("code-review.analysis.files", None, 2),
             ("code-review.analysis.lines", None, 2),
             ("code-review.issues", "source-test-coverity-coverity", 2),
-            ("code-review.issues.publishable", "source-test-coverity-coverity", 0),
+            ("code-review.issues.publishable", "source-test-coverity-coverity", 1),
```

Comment:
I'd say the build errors should be in a different statistics than normal coverity errors

Reply:
Why do you say this, since in the end they are published.

Reply:
It would be useful to be able to differentiate between statistics about Coverity issues and about build errors.

Reply:
That can be in a followup i think.

Reply:
We should do it soon though, or our stats will get biased